wireapp/wire-android
# fix(dev): message edit issues [AR-3241]

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-3241" title="AR-3241" target="_blank"><img alt="Bug" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10803?size=medium" />AR-3241</a>  PlayTest 22.03 - Edit -> Edit a mention > suggestion remains > tap on suggestion CRASH
  </td></table>
  <br />
 








----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

The message list scrolls when edit starts which can be confusing and look like the message order has been changed.
Empty spaces at the beginning or end of the edited text are not considered an edit.
The mention suggestion list doesn't disappear after the edit and can crash the app when clicking.

### Solutions

Disable scrolling to the message that's being edited (for now, when we will have the "scroll to bottom" button implemented, then it will make sense).
Allow for empty spaces at the beginning or/and the end of the edited text.
Close mention suggestion list after the edit is completed or cancelled.

### Testing

#### How to Test

Edit a message, try to add empty spaces at the beginning or/and end, try to add a mention and don't click on any user on the list, approve or cancel the edit and then see if the list closed.

### Attachments (Optional)

https://user-images.githubusercontent.com/30429749/227475220-b0063717-8417-444b-8e9c-47a879ed2e5f.mp4

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
